app/notification_email.go

```diff
@@ -100,7 +100,7 @@ func (a *App) sendNotificationEmail(notification *postNotification, user *model.
 
 	a.Srv.Go(func() {
 		if err := a.SendNotificationMail(user.Email, html.UnescapeString(subjectText), bodyText); err != nil {
-			mlog.Error(fmt.Sprint("Error to send the email", user.Email, err))
+			mlog.Error("Error while sending the email", mlog.String("E-mail", user.Email), mlog.Err(err))
```

Comment:
I would suggest to use `email` instead of `E-mail` because is a "generic" field used to handle from structured logging and is better to have consistent casing, in this fields we normally use all lowercase, and "email" all together without `-`